ReleaseProcedure.txt

```diff
@@ -55,3 +54,10 @@ Tools\gitreleasemanager\0.7.0\tools\GitReleaseManager.exe create -o nhibernate -
   * If this was a stable branch, merge it forward to master. Perhaps some
     changes need to be edited out in the merge, but this will reduce the
     risk of some issue being left unfixed on master.
+ 
+  * If this was the master branch, create the stable branch for the new release, named
+    according to its version with "x" instead of its patch number. By example, "5.3.x".
+    
+  * If next version is decided set it in NHibernate.props under build-common folder
+    (in most cases - increment minor version) and set VersionSuffix to "dev".
+    Commit it directly (without going through a PR).
```

Comment:
I think there no harm to always bump to the "next" minor, even if we end up releasing a major and not that minor. This would avoid some confusion about whether the "dev" builds are a previous pre-release or already a "vNext".
And we always need to put the "dev" sufffix back in place, in my opinion.

Reply:
>  bump to the "next" minor, even if we end up releasing a major and not that minor.

I'm not against it. But I think It might be confusing for users. They might still expect that minor version would be released along with major one.

> This would avoid some confusion about whether the "dev" builds 

As long those versions are not publicly published I think it's OK to have version mismatch for compiled libs. So 
* we shouldn't publish dev builds without "dev" suffix; 
* "dev" suffix must be set only with proper next version. 

So that way there is shouldn't be confusion with published versions. That was my initial plan.

But I'm fine with always bumping version commit plan too.